### ISSUE TITLE
"config set container-runtime cri-o" tells me to use containerD

### ISSUE THREAD

**Issue by MrBuddyCasino** [kubernetes/minikube#6011]
**The exact command to reproduce the issue**:
works: `minikube start --container-runtime=cri-o`
doesn't work: `minikube config set container-runtime cri-o`

**The full output of the command that failed**:<details>

```
💣  Set failed: [
This addon can only be enabled with the containerd runtime backend. To enable this backend, please first stop minikube with:

minikube stop

and then start minikube again with the following flags:

minikube start --container-runtime=containerd --docker-opt containerd=/var/run/containerd/containerd.sock]
```

</details>

**The output of the `minikube logs` command**:<details>

not applicable

</details>

**The operating system version**: macOS Mojave

**Comment by afbjorklund** [comment]
This was fixed in #5964

**Comment by sharifelgamal** [comment]
I'll go ahead and close this issue since it appears to be resolved. Feel free to reopen if necessary.
